doc/reference/hardware.rst

```diff
@@ -1,196 +1,100 @@
+******************
+
+When you are selecting hardware to use with ACRN, consider the
+following:
+
+* When the development team is working on a new ACRN version, we focus our
+  development and testing on one product. The product is typically a board
+  or kit from the latest processor family.
+
+* We also provide a level of maintenance for some older products.
+
+* For all products, we welcome and encourage the community to contribute support
+  by submitting patches for code, documentation, tests, and more.
+
+The following table shows supported processor families, along with the
+products that the development team has tested. The products are categorized
+into three support levels: Release, Maintenance, and Community Support. Each
+level includes the activities described in the lower levels. The noted level for
+a particular ACRN version also applies to subsequent ACRN versions not shown in
+the table.
+
+.. _NUC11TNHi5:
+   https://ark.intel.com/content/www/us/en/ark/products/205594/intel-nuc-11-pro-kit-nuc11tnhi5.html
+
+.. _NUC6CAYH:
+   https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc6cayh.html
+
+.. _NUC7i5BNH:
+   https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/NUC7i5BNH.html
+
+.. _NUC7i7BNH:
+   https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/NUC7i7BNH.html
+
+.. _NUC7i5DNH:
+   https://ark.intel.com/content/www/us/en/ark/products/122488/intel-nuc-kit-nuc7i5dnhe.html
+
+.. _NUC7i7DNHE:
+   https://ark.intel.com/content/www/us/en/ark/products/130393/intel-nuc-kit-nuc7i7dnhe.html
+
+.. _WHL-IPC-I5:
+   http://www.maxtangpc.com/industrialmotherboards/142.html#parameters
+
+.. _UP2-N3350:
+.. _UP2-N4200:
+.. _UP2-x5-E3940:
+.. _UP2 Shop:
+   https://up-shop.org/home/270-up-squared.html
+
++------------------------+------------------------------------+-------------+-------------+-------------+-------------+-------------+-------------+
+|                        |                                    |                                  ACRN Version                                     |
++------------------------+------------------------------------+-------------+-------------+-------------+-------------+-------------+-------------+
+| Intel Processor Family | Tested Product                     |   v1.0      |   v1.6.1    |    v2.0     |    v2.5     |    v2.6     |    v2.7     |
++========================+====================================+=============+=============+=============+=============+=============+=============+
+| Tiger Lake             | `NUC11TNHi5`_                      |             |             |             | Release     | Maintenance               |
++------------------------+------------------------------------+-------------+-------------+-------------+-------------+---------------------------+
+| Whiskey Lake           | `WHL-IPC-I5`_                      |             |             | Release     | Maintenance                             |

```

Comment:
Change Whiskey Lake support level to Community with v2.7. (To save space you might consider saying "Community" instead of "Community Support":
```suggestion
| Whiskey Lake           | `WHL-IPC-I5`_                      |             |             | Release     | Maintenance            | Community      |

```